elfo-configurer/src/lib.rs

```diff
@@ -101,10 +136,21 @@ impl Configurer {
         }
     }
 
-    async fn load_and_update_configs(
-        &mut self,
-        force: bool,
-    ) -> Result<(), Vec<ReloadConfigsError>> {
+    async fn check_configs(&self) -> Result<(), Vec<ReloadConfigsError>> {
```

Comment:
It should be `load_and_check_configs` in order to be consistent with `load_and_update_configs`.